engine/lib/cron.php

```diff
@@ -35,6 +35,49 @@ function _elgg_cron_init() {
 }
 
 /**
+ * Cron run handler
+ *
```

Comment:
Could you explain in the docblock, what this function is doing and why it is needed (as opposed to the other approach)?

Format:
```php
/**
 * First line
 * 
 * One or more lines, max 80 characters each. Lorem ipsum dolor sit
 * amet.
 * 
 * @access private
 */
```